server/routes/api/conversations.js

```diff
@@ -16,36 +16,36 @@ router.get("/", async (req, res, next) => {
       where: {
         [Op.or]: {
           user1Id: userId,
-          user2Id: userId,
-        },
+          user2Id: userId
+        }
       },
       attributes: ["id"],
-      order: [[Message, "createdAt", "DESC"]],
+      order: [[Message, "createdAt", "ASC"]],
       include: [
-        { model: Message, order: ["createdAt", "DESC"] },
+        { model: Message, order: ["createdAt", "ASC"] },
         {
```

Comment:
The problem with doing this (alone) is that now the order of the conversations is wrong, and the text that shows the most recent message now displays the last! Please update this solution to ensure the order of conversations and message preview text are not broken 🙂